<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EMBYAggregationSearchClass</class>
 <widget class="QMainWindow" name="EMBYAggregationSearchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EMBYAggregationSearch</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QComboBox" name="SearchSelection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>521</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SingleSearch">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>360</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Single Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GlobalSearch">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Global Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SearchText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>631</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="FliterSelection">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>50</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtCurrent</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <item>
       <property name="text">
        <string>ALL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Series</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Movie</string>
       </property>
      </item>
     </widget>
     <widget class="QTableWidget" name="SearchResult">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>671</width>
        <height>241</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Server</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Link</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>添加服务器</string>
      </property>
      <widget class="QLineEdit" name="ServerURL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>ServerURL Example:https://emby.bilibili.com</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Usertoken">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>361</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Usertoken Example:123abc</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AddServer">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>ADD Server</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>591</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>登录获取Token</string>
      </property>
      <widget class="QLineEdit" name="Login_ServerURL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>ServerURL Example:https://emby.bilibili.com</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Login_Username">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Login_Password">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Password(Wont be Saved)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Login">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>50</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>LOG IN</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Login_Token">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>201</width>
         <height>81</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EMBYAggregationSearch.qrc"/>
 </resources>
 <connections/>
</ui>
